wrappers/tests/dem/test_kratos_dem.py

```diff
@@ -0,0 +1,67 @@
+""" test_kratosWrapper module
+
+This module contains the unitary tests for the
+kratosWrapper class.
+
+"""
+import sys
+import os
+
+sys.path.append(os.path.abspath('wrappers/tests/dem'))
```

Comment:
This causes problem when trying to run the tests outside of ```simphony-kratos```.  I will open up an issue to follow this.